SubEthaEdit-Mac/Source/PlainTextDocument.m

```diff
@@ -2390,7 +2390,20 @@ - (void)TCM_ensureFileTypeDataOrSEEText {
 - (void)saveDocumentWithDelegate:(id)delegate didSaveSelector:(SEL)didSaveSelector contextInfo:(void *)contextInfo {
     if ([self TCM_validateDocument]) {
```

Comment:
style: please do reindent the whole function with ctrl-I if we leading whitespace on some of the lines.